mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -309,7 +311,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 setActionBarTitle(R.string.create_group);
                 break;
             case R.id.logout:
-                logout();
+                final AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
@pranavgupta1234, Please use this [MaterialDialog](https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MaterialDialog.java) class.